app/api/apiConfig.js

```diff
@@ -128,6 +128,317 @@ export const settingsAPIs = {
+        {
+            url: "wss://singapore.walldex.pro/ws",
+            region: "Southeastern Asia",
+            country: "Singapore",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://southafrica.walldex.pro/ws",
+            region: "Southern Africa",
+            country: "South Africa",
+            location: "Johannesburg",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://australia.walldex.pro/ws",
+            region: "Australia and New Zealand",
+            country: "Australia",
+            location: "Sydney",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://au-wa.walldex.pro/ws",
+            region: "Australia and New Zealand",
+            country: "Australia",
+            location: "Perth",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://newzealand.walldex.pro/ws",
+            region: "Australia and New Zealand",
+            country: "New Zealand",
+            location: "Auckland",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://brazil.walldex.pro/ws",
+            region: "South America",
+            country: "Brazil",
+            location: "São Paulo",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://chile.walldex.pro/ws",
+            region: "South America",
+            country: "Chile",
+            location: "Curico",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://argentina.walldex.pro/ws",
+            region: "South America",
+            country: "Argentina",
+            location: "Buenos Aires",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://colombia.walldex.pro/ws",
+            region: "South America",
+            country: "Colombia",
+            location: "Bogota",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://turkey.walldex.pro/ws",
+            region: "Western Europe",
+            country: "Turkey",
+            location: "Istanbul",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://israel.walldex.pro/ws",
+            region: "Western Europe",
```

Comment:
Can you please double-check all regions according to above list `nodeRegions`. The link there shows the classification (for easy "local" geolocation)